integrations/amazon_bedrock/tests/test_text_embedder.py

```diff
@@ -76,13 +77,17 @@ def test_from_dict(self, mock_boto3_session):
                 "aws_profile_name": {"type": "env_var", "env_vars": ["AWS_PROFILE"], "strict": False},
                 "model": "cohere.embed-english-v3",
                 "input_type": "search_query",
+                "boto3_config": {
```

Comment:
same thing as before

Reply:
Done. I parametrized the tests for the boto3_cofig value.